.github/label-actions.yml

```diff
@@ -20,4 +20,42 @@ Needs Logs:
 
     3. Get the log file and attach it to this issue. By default this will be in `%TMP%/LogFiles/Application/Functions/Host`.
 
+    **NOTE** Debug logging will include information such as Database and table names, if you do not wish to include this information you can either redact it from the logs before attaching or let us know and we will provide a way to send logs directly to us.
+
+#actions for Needs Connection Logs
+Needs Connection Logs:
+  comment: |
+    We need more info to debug your particular issue. If you could attach your SQL Client logs to the issue, it would help us fix the issue much faster.
+
+    1. Set the AzureFunctions_SqlBindings_VerboseLogging to `true` in your `*.settings.json` file or your Function App settings.
+
+    ```json
+    {
+    "IsEncrypted": false,
+    "Values": {
+      "AzureWebJobsStorage": "UseDevelopmentStorage=true",
+      "FUNCTIONS_WORKER_RUNTIME": "<dotnet>",
+      "SqlConnectionString": "<Your Connection string>",
+      "AzureFunctions_SqlBindings_VerboseLogging": "true",
+    }
+    }
+    ```
+
+    2. Set the default logging level to `Trace` and enable logging to a file. To do this ensure the following entries are in your `host.json`. ([instructions](https://learn.microsoft.com/azure/azure-functions/configure-monitoring?tabs=v2#configure-log-levels))
+
+    ```json
+    {
+      "logging": {
+        "fileLoggingMode": "always",
+        "logLevel": {
+          "default": "Trace"
+        }
+      }
+    }
+    ```
+
+    3. Restart your function and reproduce your issue
+
+    4. Get the log file and attach it to this issue. By default this will be in `%TMP%/LogFiles/Application/Functions/Host`.
```

Comment:
Actually, looking at this it would be good to include instructions (for both this and the other logging message) for doing this in a deployed function too since they won't be the same. Not sure offhand what the easiest way to export the logs would be, so that might take some investigation.

In the meantime I think we can actually revert the change I had you add earlier about the Function App settings and instead specify that these instructions are for local development only. Then when we figure out a way for deployed functions we can come back and update both with those instructions.

Reply:
Thanks for adding the note! Might be worth making an actual issue to track updating this with instructions for deployed functions as well. 

(although if we do that it might actually be best to just update the instructions in our docs and then link to those from here so we don't have to remember to keep both in sync)